packages/@azure/servicebus/data-plane/test/renewLockSessions.spec.ts

```diff
@@ -39,10 +43,6 @@ describe("Standard", function(): void {
         await beforeEachTest(receiverClient);
       });
 
-      after(async () => {
```

Comment:
how about using afterEach?
Also, did we run the tests? check my latest comment on the issue - I'm not sure if these changes address the root cause.

Reply:
> how about using afterEach?
Namespaces can be safely re-used across tests. But the senderClient and receiverClient shouldnt be as each test should work on its own sender/receiver links so as to not affect other tests

> Also, did we run the tests?
Yes, we (Harsha and I) ran the tests multiple times. For the first time **ever** (after we first started seeing the OperationTimeoutError), all renew lock related tests are passing successfully

> check my latest comment on the issue - I'm not sure if these changes address the root cause
I have addressed your comment in https://github.com/Azure/azure-sdk-for-js/pull/1120#issuecomment-460818592

Reply:
We should also ideally update all other test files to avoid closing the namespace after each tests. I believe this affected the renewlock tests the most because the clients were not being re-created.